### DISCUSSION TITLE
[Projects Beta] Make tasks (drafts) linkable objects

### DISCUSSION THREAD

**Discussion by ashleyflorence** [community/community#16610] [Projects and Issues]
Can we make the drafts in the projects linkable - this way we can reference tasks set out within the project.

In a project we may have several tasks assigned to different folks, but there are some tasks that lean and depend on each other and it would be beneficial to be able to use a link to reference the other task. ie., Task A and Task G are related so in their descriptions it we could have the links to each other

**Comment by LydiaWilliams** [comment]
Yes! This would be really useful for sharing a draft issues within a team for reviewing drafts before they are converted to a full-fledged issue.

**Comment by bplatta** [comment]
This is one of the biggest annoyances for our team. I'd like to be able to send an engineer a link to a specific drafted issue.

Edit: I realize my ask is actually for a different (and simpler) feature than the linking described here which is allowing tasks to be related to other tasks. I just want a url to drafted issue.

**Comment by appleseed-iii** [comment]
This is a big pain point right now.
`+ 1000`

**Comment by mdrabic** [comment]
This is similar to https://github.com/github-community/community/discussions/15611

**Comment by vvo** [comment]
I really like GitHub projects but this is the kind of issue that could make us switch to a different project tracking product.

Moreover, there should be no difference between tasks and issues in terms of features. Easy to say, but, naively I see tasks as unpublished issues that's it.

I should be able to do all of the actions from a GitHub issue in a task. The only difference being that tasks are hidden in the list of issues.

**Comment by chrisglein** [comment]
This is one of the limitations of the new Projects system relative to the old boards. "Card" items could be linked to ("Copy card link"). "Draft" items cannot. With this implemented I could migrate some of our old boards to the new system with basically no downside.

**Comment by Malix-Labs** [comment]
How about being able to build "draft" issues, as an issue object, not only for project?

**Reply by Malix-Labs** [reply]
#19498

**Comment by thomas-fred** [comment]
This would be great! :+1:

**Comment by mattcosta7** [comment]
https://github.blog/changelog/2023-01-05-github-issues-january-5th-update/

We recently shipped urls that link directly to the projects sidepanel open to an issue or draft issue

**Reply by ashleyflorence** [reply]
I saw this! 

Thank you for making such an impactful change!!

**Reply by itcarroll** [reply]
Used this new feature today, thanks! I think going further with @Malix-Off's [19498](https://github.com/orgs/community/discussions/19498) is still essential. Linking to a pane doesn't give you, for instance, the ability to add a draft to a task list. Also see [41798](https://github.com/orgs/community/discussions/41798).
